src/pepr/operator/reconcilers/index.ts

```diff
@@ -70,7 +74,7 @@ export async function updateStatus(cr: UDSPackage, status: PkgStatus) {
  * @param type The type of event to write
  */
 export async function writeEvent(cr: UDSPackage, event: Partial<kind.CoreEvent>) {
-  Log.debug(cr.metadata, `Writing event: ${event.message}`);
+  log.debug(cr.metadata, `Writing event: ${event.message}`);
```

Comment:
I think here we can/should find a way to clean up this log (it looks a bit messy currently logging out the full metadata). We can probably just log out the name/namespace?